requirements-optional.txt

```diff
@@ -1,4 +1,4 @@
 matplotlib
 pyrevolve==2.2.4
-scipy>=1.13.0
+scipy
```

Comment:
I think it is better to add an upper bound:

e.g. <=

and then (if needed) do what  @mloubout did here (?):

https://github.com/devitocodes/devito/commit/dee09dda5ad89c72d82ed9a84a839a1ccbbdb6ba

Reply:
keep lower bound ofc